libs/SalesforceSDK/src/com/salesforce/androidsdk/developer/support/notifications/local/ShowDeveloperSupportNotifier.kt

```diff
@@ -93,8 +97,9 @@ internal class ShowDeveloperSupportNotifier {
         /**
          * Displays the Salesforce Mobile SDK "Show Developer Support"
          * notification.
-         * * @param activity The Android activity
+         * @param activity The Android activity
          */
+        @SuppressLint("LaunchActivityFromNotification")
```

Comment:
This suppress is what the trigger activity aided in avoiding.  Thinking on it a bit longer though, our difference from the use case described by the Material design docs makes this suppress logical.  This notification doesn't have a dedicated activity to display as Material suggests - it's triggering an action in the visible activity.